tests/unit/XMLDataFormatterTest.php

```diff
@@ -74,4 +76,71 @@ public function testConvertStringToArrayMultipleEntitiesException()
         $formatter = new XMLDataFormatter();
         $formatter->convertStringToArray($inputXML);
     }
+
+    /**
+     * Tests wrapper output of {@link XMLDataFormatter::convertDataObjectWithoutHeader()}
+     */
+    public function testConvertDataObjectWithoutHeaderClassNameAttribute(): void
```

Comment:
These two tests are basically identical, use a single test and use a `@dataProvider` and a `bool $trainingSlash` param.

Reply:
Yep, I was being lazy 😪